### PR TITLE
feat(Chart): Added Performance Chart Component

### PR DESCRIPTION
**Pull request by sandeep-v1404**
Signed-off-by: sandeep-v1404 <valpasanisandeep@gmail.com>

### Functionality:

To show Performance Chart in Dashboard . And also to query with custom sprint range.

### Solution:

To implement chart using Google Charts. And also to make low level components.

### Risk level:
- [X] high 
- [ ] medium
- [ ] low

### How to test:

- ng serve
- In main Page, left side there is performance chart .
- Query using the top right icon to know your performance in previous sprints.

### 

Reference:

1.Desktop/PC/Laptop view
my-dashboard

2.Mobile View
my-dashboard-mobile

### CODE REVIEW

**Comment by Shrishti1802** [comment]
Resolve merge conflicts

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
In validation some extra changes occur so please undo that changes

File: worktrolly-ui/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "worktrolly-ui",
-  "version": "0.0.0",
+  "name": "worktrolly",
+  "version": "0.2.0",
```
[the remark this answers is not in the document]
Comment (sandeep-v1404) [reply worktrolly-ui/package-lock.json]: > In validation some extra changes occur so please undo that changes

Formatting was done properly by @k0malSharma's branch -> validation service file . Now it was fixed Correctly.

[the remark this answers is not in the document]
Comment (shivamgoyal7) [reply worktrolly-ui/package-lock.json]: ok

**Review by shivamgoyal7 — APPROVED** [review]
ship it

File: worktrolly-ui/package.json [hunk 2]
```diff
@@ -20,13 +20,14 @@
...
+        "angular-google-charts": "^2.0.0",
         "bootstrap": "^4.5.3",
+        "firebase": "^7.0.0 || ^8.0.0",
         "jquery": "^3.5.1",
         "popper.js": "^1.16.1",
         "rxjs": "~6.6.0",
         "tslib": "^2.0.0",
-        "zone.js": "~0.10.2",
-        "firebase": "^7.0.0 || ^8.0.0"
+        "zone.js": "~0.10.2"
```
Comment (viv9k) [inline worktrolly-ui/package.json:30]: Revert these changes

**Review by viv9k — APPROVED** [review]
Ship it

**Review by viv9k — CHANGES_REQUESTED** [review]
Please complete the required changes

**Comment by sandeep-v1404** [comment]
> Please complete the required changes

Yes , It was done automatically according to alphabetical order.

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**